packages/php-wasm/universal/src/lib/php-request-handler.ts

```diff
@@ -379,15 +378,10 @@ function seemsLikeADirectoryRoot(path: string) {
 export function applyRewriteRules(path: string, rules: RewriteRule[]): string {
 	for (const rule of rules) {
 		if (new RegExp(rule.match).test(path)) {
-			if (rule.keep) {
-				path = path.replace(
-					path.replace(new RegExp(rule.keep), ''),
-					''
-				);
-			}
-			if (rule.remove) {
-				path = path.replace(new RegExp(rule.remove), '');
+			if (rule.replacement.startsWith('$')) {
```

Comment:
Replacement could be `abc$2`, let’s just alwaya call .replace()

Reply:
Done [c928110](https://github.com/WordPress/wordpress-playground/pull/1083/commits/c92811033207c07bff27b388b5282e0faa971c82)